### DISCUSSION TITLE
Working of email deep linking when app isn't installed

### DISCUSSION THREAD

**Discussion by muninderkaur06** [lexiconhq/lexicon#56] [Troubleshooting and Q&A] [answered]
Hi there,

I've been using your project for a long time now & I'm happy with the progress you guys are making. 

I was just curious about your : https://github.com/lexiconhq/lexicon/issues/15, where you guys have mentioned 'And if the app isn’t installed, or the user is not on a mobile device, it would pass them through to the site as usual.'. How and where have you configured this part?

**Comment by Kinostrome** [comment, marked as answer]
Hello,

For these kinds of topics, we'd always recommend reading the documentation, which contains more up-to-date information than previous closed issues.

You can read more about email deep linking here: https://docs.lexicon.is/version-2.0.0/email-deep-linking/discourse-plugin/email-deep-linking
